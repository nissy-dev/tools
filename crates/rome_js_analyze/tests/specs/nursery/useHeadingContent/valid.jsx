<>
	<h1>heading</h1>
	<h1>
		<div aria-hidden="true"></div>visible content
	</h1>
	<h1 dangerouslySetInnerHTML={{ __html: "heading" }} />
	<h1>
		<div aria-hidden />
		visible content
	</h1>
	<h1 dangerouslySetInnerHTML={{ __html: "heading" }}></h1>
</>;
