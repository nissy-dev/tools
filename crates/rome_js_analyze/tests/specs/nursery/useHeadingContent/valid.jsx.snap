---
source: crates/rome_js_analyze/tests/spec_tests.rs
expression: valid.jsx
---
# Input
```js
<>
	<h1>heading</h1>
	<h1>
		<div aria-hidden="true"></div>visible content
	</h1>
	<h1 dangerouslySetInnerHTML={{ __html: "heading" }} />
	<h1>
		<div aria-hidden />
		visible content
	</h1>
	<h1 dangerouslySetInnerHTML={{ __html: "heading" }}></h1>
</>;

```


